python/lsst/ts/hvac/__init__.py

```diff
@@ -27,4 +27,6 @@
 from .config_schema import CONFIG_SCHEMA
 from .hvac_csc import *
 from .hvac_enums import *
+from .hvac_utils import *
```

Comment:
This duplication of "hvac" in these modules names is a bit unfortunate, as it leads to stutter: `lsst.ts.hvac.hvac_foo`

Reply:
An old habit from Java where class files need to have the same name as the class they implement. I renamed all three files and dropped the `hvac_` prefix.